content/docs/buildpack-author-guide/create-buildpack/caching.md

```diff
@@ -23,7 +23,8 @@ echo "---> Installing gems"
 bundlerlayer="$layersdir/bundler"
 mkdir -p "$bundlerlayer"
 echo -e 'cache = true\nlaunch = true' > "$bundlerlayer.toml"
-bundle install --path "$bundlerlayer" --binstubs "$bundlerlayer/bin"
+bundle config set --local path "$bundlerlayer" && bundle install && bundle binstubs --all --path "$bundlerlayer/bin"
```

Comment:
I noticed this while running the buildpack. Looks like `--path` and `--binstubs` have been deprecated. I moved them to their appropriate replacements.